docs/javascript.md

```diff
@@ -88,6 +88,32 @@ Takes these options:
 
 Full documentation on how to design multivariate tests, use the data in GA and construct hypothesis tests is on its way soon.
 
+
+## Reporting to Google Content Experiments
+The toolkit includes a library for multivariate testing that is capable of reporting data into [Google Content Experiments](https://developers.google.com/analytics/devguides/platform/experiments-overview). 
+
+### To create a new experiment
+
+1. Log in to Google Universal Analytics.
+2. Follow [these instructions](https://support.google.com/analytics/answer/1745152?hl=en-GB) to set up your experiment; you will need to have edit permissions on the Universal Analytics profile. If you don't have them, you can ask someone from the Performance Analyst team to set the experiment up for you.
+3. In step number 3, "Setting up your experiment code", select "Manually insert the code" and make a note of the Experiment ID number located under the script window.
+4. Add the below code to the page you want to test.
+    - the contentExperimentId is the Experiment ID you retrieved in step 3 
+    - the variantId is 0 for the original variant, 1 for the first modified variant, 2 for the second modified variant, etc.
+
+```js
+var test = new GOVUK.MultivariateTest({
```

Comment:
Does this code block need to be added before or after the analytics setup, or does it not matter?